pipelines/pipelines/obr-generic/build-branch.yaml

```diff
@@ -348,7 +348,7 @@ spec:
     when:
       - input: "$(params.toBranch)"
         operator: notin
```

Comment:
The operator here is 'notin' so you are saying OBR Generic **should not trigger** Eclipse if this is for 'main' **or** 'release'. This means Eclipse won't really ever be called. Did you mean to do this?